docker-files/base/docker-bootstrap-15.1.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+for option in "$@"
+do
+  case "${option}" in
+  frontend)
+    zypper -n install --no-recommends --replacefiles \
```

Comment:
this won't fly. We want the containers to build in OBS - but OBS parses the Dockerfile to understand what packages are needed and zypper commands are magic. Hiding them in shell scripts won't work.

Reply:
Our development environment we are building and uploading it to dockerhub. It's not planned to do it in this PR, but we can and should do it later. So I would handle it as nitpick

Reply:
I won't because in test development we build in OBS - and we should stop using the old way and you make it *worse*.

Reply:
why do we make it worse using dockerhub @coolo

Reply:
for once: because SUSE as a company moves away from docker (towards podman and it's own registry). But that's not the main problem  - the main problem is the unifaction of test containers and dev containers. And those live in obs.

Reply:
You didn't answer why I'm making it technically worse with this PR, updating it from 42.3 to 15.1 @coolo.

Reply:
well, ok - you're not making it worse (beside the obvious duplication of most files), but you're not improving the situation either.

Reply:
Hey @coolo. Thank you for the clarification. Actually I'm not duplicating any file, I'm adding support to 15.1 without remove the support to 42.3. That's necessary until we are sure that everything is working properly, then we can remove all 42.3 files as soon as we stick with 15.1. Is that ok for you or do you think that we have to radically drop support for 42.3 loosing the change to compare both environments if extra debug is needed?

Reply:
@coolo, 42.3 goes EOL in the end of the month. How moving our development environment to the last maintained leap doesn't improve the situation?

Reply:
If you want to keep the 42.3 containers, tag them on dockerhub